src/screens/menu/components/MenuButton.tsx

```diff
@@ -33,4 +35,8 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'center',
   },
+  menuBoxWrap: {
+    minWidth: 200,
+    flexWrap: 'nowrap',
```

Comment:
I don't think this is doing anything - at least on Android.

Reply:
Removed the flexWrap: 'nowrap',